tsl/test/sql/chunk_merge.sql

```diff
@@ -29,16 +29,16 @@ SELECT * FROM _timescaledb_catalog.chunk;
 \set ON_ERROR_STOP 0
 
 -- Cannot merge chunks from different hypertables
-SELECT _timescaledb_internal.test_merge_chunks_on_dimension('_timescaledb_internal._hyper_1_1_chunk','_timescaledb_internal._hyper_3_7_chunk', 1);
+SELECT _timescaledb_internal.test_merge_chunks_on_dimension('_timescaledb_internal._hyper_1_1_chunk','_timescaledb_internal._hyper_3_13_chunk', 1);
```

Comment:
Seems odd that these need to be updated each time we change anything in the code. Can we make this generic?

Reply:
I don't think its every time we change anything in the code, we have to use multiple specific chunks to test this functionality and generalizing them would be more work than the actual worth.

Changes here are infrequent enough to absorb the eventual cost of patching the test with new chunk names.